src-v2/Not/Not/Extensions/DateTimeOffsetExtension.cs

```diff
@@ -13,11 +14,10 @@ public static DateTimeOffset ToDateTimeOffset(this DateTime time)
         DateTimeOffset offsetTime = timeWithSpecifiedKind;
         return offsetTime;
     }
-    public static DateTimeOffset? ToDateTimeOffset(this TimeSpan? timeToBeAdded)
+    public static DateTimeOffset ToDateTimeOffset(this TimeSpan timeToBeAdded)
     {
-        if(timeToBeAdded == null) return null;
         DateTime today = DateTime.Today;
-        var time = today.Add((TimeSpan)timeToBeAdded);
+        DateTime time = today.Add(timeToBeAdded);
         DateTime timeWithSpecifiedKind = DateTime.SpecifyKind(time, DateTimeKind.Local);
         DateTimeOffset offsetTime = timeWithSpecifiedKind;
```

Comment:
Implicit vars again, It seems you have a habit of using them (or is it the IDE somehow? ). Try to be more vigilant and replace then with `var` whenever you see them, until you make a habit of using it in the first place.